### PR TITLE
Added a foundation to README.md to build upon

### PR DESCRIPTION
**Pull request by bnb**
Added basic structure and information to the README.md, as well as a basic form to submit as an Issue.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,43 @@
...
+* Enterprise
+* Individual
+* Student
+* Active Contributor
+
+## What are the conditions of eligibility?
+
+TBD - Join the discussion on this at [issues/8](https://github.com/nodejs/membership/issues/8)
+
+## How to join
```
Comment (Morgul) [inline README.md:14]: As I mentioned [here](https://github.com/nodejs/membership/issues/6#issuecomment-107222887), I believe the GH issues are only for running for the board seat, not for joining the foundation. That should be done via a form on the website, and a paypal link, I would assume.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,43 @@
...
+
+To apply to be a member of the Node Foundation, create a new issue in this repository. Use the formatting given below, and preview your post before you submit it to make sure formatting and content are correct.
+
+### Application structure:
```
Comment (Morgul) [inline README.md:18]: I'm a little uncomfortable with the idea of having an application for membership; provided I'm willing to give the foundation the requested amount of money per year, I should be able to join, quite literally with no questions asked.

Comment (bnb) [inline README.md:18]: That's fair play - I thought the goal with application was to simply ensure members have an intent to develop, maintain, and, more generally, contribute to the Node community. Could be wrong on that, though. Let's keep this discussion in https://github.com/nodejs/membership/issues/6, to make sure all points are represented at a single location.

I have quoted your comment in https://github.com/nodejs/membership/issues/6 to maintain the integrity of the conversation.

**Comment by ashleygwilliams** [comment]
this seems stale and also somewhat redundant with #13, close in favor of filing an issue to add any info here that #13 missed?

File: README.md (hunk 1 above)
Comment (mikeal) [inline README.md:14]: There's actually just three "classes" of membership: corporate, associate and individual.

Student, Active Contributor, etc are all "Individual Members."

I would talk about the difference between corporate or individual, ignore associate (which is reserved for universities and non-profits, we don't actually have any of these yet).

File: README.md (hunk 2 above)
Comment (mikeal) [inline README.md:18]: You just signup, there's no application.

**Comment by mikeal** [comment]
Oh wow, didn't realize how old this was when i started commenting on it.

**Comment by ashleygwilliams** [comment]
heh yeah, @mikeal tis why i suggested closing :wink: 
also, after review, i don't think this has anything we don't already have.

**Pull request closed without merging**